src/infrastructure/api/index.ts

```diff
@@ -113,20 +114,25 @@ router.get(
   ) => {
     let secretsManager: SecretsManagerServiceInterface;
     if (process.env.IS_OFFLINE === 'true') {
+      logger.debug('configuring local secret manager');
       secretsManager = new LocalSecretsManagerService();
     } else {
+      logger.debug('configuring aws secret manager');
       secretsManager = new SecretsManagerService(new AWS.SecretsManager());
     }
     const fileName = `EVL_GVT_${moment(Date.now()).format('YYYYMMDD')}.csv`;
+    logger.debug(`creating file for EVL feed called: ${fileName}`);
     DatabaseService.build(secretsManager, mysql)
       .then((dbService) => getEvlFeedDetails(request.query, evlFeedQueryFunctionFactory, dbService))
       .then((result) => {
-        console.log('Generating EVL File Data');
+        logger.info('Generating EVL File Data');
         const evlFeedProcessedData: string = result.map(
           (entry) => `${entry.vrm_trm},${entry.certificateNumber},${moment(entry.testExpiryDate).format('DD-MMM-YYYY')}`,
         ).join('\n');
+        logger.debug(`\nData captured for file generation: ${evlFeedProcessedData} \n\n`);
 
         uploadToS3(evlFeedProcessedData, fileName, () => {
+          logger.debug(`Successfully uploaded ${fileName} to S3`);
```

Comment:
I think this would be a good as info rather than debug?

Reply:
Can you add it as a suggestion please 😄

Reply:
Done